src/System Application/App/Language/src/AllowedLanguages.Page.al

```diff
@@ -18,7 +18,7 @@ page 3563 "Allowed Languages"
     HelpLink = 'https://go.microsoft.com/fwlink/?linkid=2149387';
     AdditionalSearchTerms = 'company,role center,role,language';
     AboutTitle = 'About allowed languages.';
-    AboutText = 'Define a list of allowed languages which is enabled for this environment. If nothing is specified, then the user will be able to use all available languages.';
+    AboutText = 'Define a list of allowed languages which is enabled for this environment. If nothing is specified, the user will be able to select from all languages.';

```

Comment:
Small change, sorry.

Before: "...which is enabled for this environment."
New: "... which **are** enabled **in** this environment."